jwst/persistence/persistence.py

```diff
@@ -302,9 +310,10 @@ def do_all(self):
         # file are flagged as bad, set the corresponding data values to
         # zero, so the computed number of traps will also be zero.
         if hasattr(self.trap_density, "dq"):
-            mask = (np.bitwise_and(self.trap_density.dq,
-                                   dqflags.pixel["DO_NOT_USE"]) > 0)
-            self.trap_density.data[mask] = 0.
+            mask = (
+                np.bitwise_and(self.trap_density.dq, dqflags.pixel["DO_NOT_USE"]) > 0
+            )  # fmt: skip
```

Comment:
As in other similar cases - skipping formatting is not helping readability here.
```suggestion
            mask = np.bitwise_and(self.trap_density.dq, dqflags.pixel["DO_NOT_USE"]) > 0
```